app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background"
+    tools:context=".FragmentMoviesList">
+
+    <TextView
+        android:id="@+id/tv_movie_list"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:background="@color/background"
+        android:drawablePadding="10dp"
+        android:text="@string/movies_list"
+        android:textColor="@color/white"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:drawableStartCompat="@drawable/shape" />
+
+    <ImageView
```

Comment:
@sanya-arsenal The approach with a big image view not very flexible in this case. Please add another ViewGroup and put all list item stuff inside it. And add a click listener to this ViewGroup.